src/generate/interrupt.rs

```diff
@@ -176,7 +189,7 @@ pub fn render(
         }
     };
 
-    if target == Target::CortexM || target == Target::Msp430 {
+    if target == Target::CortexM {
         root.extend(interrupt_enum);
     } else {
```

Comment:
Won't this `else` branch still trigger, causing `#interrupt_enum` to end up in `mod_items` along with the `try_from` impl etc?

Reply:
Good catch. I wonder why this didn't cause any CI problems...